Test/Test/ConsoleReader.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <conio.h>
+#include <string>
+
+#include "ConsoleReader.h"
+
+#include <log4cpp/Portability.hh>
+#include <log4cpp/PropertyConfigurator.hh>
+#include <log4cpp/Category.hh>
+#include <log4cpp/Appender.hh>
+#include <log4cpp/FileAppender.hh>
+#include <log4cpp/OstreamAppender.hh>
+#include <log4cpp/Layout.hh>
+#include <log4cpp/BasicLayout.hh>
+#include <log4cpp/Priority.hh>
+
+int ConsoleReader::readConsole(std::string const &message) {
+	log4cpp::Category& reader =
```

Comment:
Why do we need to create the instance in each method we are going to have in this class? Maybe better is to have one log instance for whole class instance?

Reply:
Yes, that makes more sense, thank you